### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SerhiiUnhurian**
- [DEMO LINK](https://SerhiiUnhurian.github.io/layout_catalog/)
- [TEST REPORT LINK](https://SerhiiUnhurian.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,360 @@
...
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item" ddata-qa="nav-hover">
```
Comment (Esceype) [inline src/index.html:35]: Wrong data attribute and move it to nav link not to item

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,360 @@
...
+            <p class="card__reviews-count">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price-container">
+            <p class="card__price-name">Price:</p>
+            <p class="card__price-price">$2,199</p>
+          </div>
+          <div data-qa="hover">
```
Comment (Esceype) [inline src/index.html:133]: This data-qa attribute should be on link

File: src/index.html [hunk 3]
```diff
@@ -4,9 +4,360 @@
...
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container">
```
Comment (Esceype) [inline src/index.html:63]: Why do you need another one block inside main if you can move this class to main tag ?

File: src/styles/blocks/catalog/card.scss [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  grid-template-columns: repeat(4, 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  grid-auto-flow: row;
+  justify-content: center;
+  align-content: center;
+
+  justify-items: center;
+  align-items: center;
+  margin-block: 50px 72px;
```
Comment (Esceype) [inline src/styles/blocks/catalog/card.scss:12]: Checklist 8

File: src/styles/blocks/catalog/card.scss [hunk 5]
```diff
@@ -0,0 +1,100 @@
+.container {
```
Comment (Esceype) [inline src/styles/blocks/catalog/card.scss:1]: Checklist 9

**Review by Esceype — CHANGES_REQUESTED** [review]
At all great job!!! But at the moment your tests doesn`t work, so redeploy your project and fix the data-qa attributes

File: src/styles/blocks/catalog/card.scss (hunk 4 above)
Comment (SerhiiUnhurian) [reply src/styles/blocks/catalog/card.scss:12]: I know, but I don't understand how to add mix here(

File: src/styles/blocks/reset.scss [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+html {
+  font-family: "Roboto", "Arial", sans-serif;
+}
+
+body {
+  margin: 0;
+  // --button-color: #00acdc;
+  // --main-color: #060b35;
+  // --secondary-color: #616070;
+  // --background-color: #fff;
```
Comment (xiimiik) [inline src/styles/blocks/reset.scss:18]: delete comments before PR

**Review by xiimiik — APPROVED** [review]
Nice work!

**Pull request closed without merging**